portal/portal-impl/impl/src/bundle/sitenav.properties

```diff
@@ -52,6 +52,8 @@ sit_edit = Edit
 sit_reset = Tool Home
 sit_unpublished = Unpublished Site
 sit_publish_now = Publish Now
+sit_publish_confirmation = The site and its contents will be visible by the members, are you sure?
```

Comment:
This is a comma splice. Please change the comma to a period and capitalize the "A" in "are". 

Also, with the heading of this dialog as "Unpublished Site" even though it is about the publishing action, I think we should change the question to "Are you sure you want to publish this site?".